modules/prebidServerBidAdapter/index.js

```diff
@@ -822,9 +822,9 @@ const OPEN_RTB_PROTOCOL = {
           bidObject.meta = bidObject.meta || {};
           if (bid.adomain) { bidObject.meta.advertiserDomains = bid.adomain; }
 
-          // TODO: Remove when prebid-server returns ttl and netRevenue
           const configTtl = _s2sConfig.defaultTtl || DEFAULT_S2S_TTL;
-          bidObject.ttl = (bid.ttl) ? bid.ttl : configTtl;
+          // the OpenRTB location for "TTL" as understood by Prebid.js is "exp" (expiration).
+          bidObject.ttl = (bid.exp) ? bid.exp : configTtl;
```

Comment:
This works.

Although I would just do 

```js
bidObject.ttl = bid.exp || configTtl;
```

But maybe I am not thinking of some use case as the previous code was doing the same thing.